core/src/main/java/contacts/core/log/LoggerRegistry.kt

```diff
@@ -0,0 +1,30 @@
+package contacts.core.log
+
+import contacts.core.CrudApi
+import contacts.core.Redactable
+import contacts.core.redactedCopyOrThis
+
+class LoggerRegistry @JvmOverloads constructor(
+   private val logger: Logger = EmptyLogger(),
```

Comment:
We don't really need to have a default value for this as we construct this elsewhere anyways.

Reply:
I made the update here https://github.com/vestrel00/contacts-android/commit/e321874e5c37739a6cb0a26ac479400a662b911e